Misc/NEWS.d/next/Library/2020-05-18-15-38-25.bpo-25920.PxrLY8.rst

```diff
@@ -0,0 +1,5 @@
+On macOS, :func:`socket.getaddrinfo` no longer uses an internal lock to
```

Comment:
The patch also removes the lock for FreeBSD, OpenBSD en NetBSD.  Also note that the lock is only used when Python is build for an old enough version of these operating systems.  In particular, the removed code was already not used in the installers for macOS on Python.org.

Reply:
I updated the NEWS entry to mention that python.org installer wasn't affected, and mention NetBSD, OpenBSD and FreeBSD. Does the updated NEWS entry look better?